### PR TITLE
Bulma hamburger navbar

### PR DESCRIPTION
**Pull request by Talha5**
Solves ui enhancement #50

### CODE REVIEW

**Comment by phallstrom** [comment]
@Talha5 Sweet!  It mostly works and I'm not sure if it's Safari or Bulma or we're missing a class name.  What I'm seeing is...

- wide browser, see menu
- narrow browser, see hamburger, and menu stacks (I can still see the two menu items).
- click hamburger, menu closes, hamburger is replaced with an X
- click X, menu is visible (stacked), X is replaced by hamburger.

It's like the icon is doing the opposite of what it should?  Are you seeing the same?

screen shot 2018-10-13 at 9 42 02 am
screen shot 2018-10-13 at 9 42 15 am

File: app/views/layouts/_nav.html.slim [hunk 1]
```diff
@@ -4,8 +4,11 @@
       .navbar-item
         = image_tag "railslink-icon.png", alt: "", id: "logo"
         b= configatron.app_name
+      a.navbar-burger.burger aria-expanded="false" aria-label="menu" data-target="navbarBasicExample" role="button"
```
Comment (phallstrom) [inline app/views/layouts/_nav.html.slim]: Is the data-target attribute necessary?

Comment (Talha5) [reply app/views/layouts/_nav.html.slim]: Yeah, that has to link with navbar-menu class.

**Comment by Talha5** [comment]
@phallstrom I think this should solve the problem.

**Comment by phallstrom** [comment]
@Talha5 Working great!  And now that it works I notice that the hamburger/X and nav links (when stacked) are the default blue for bulma.  Would you mind overriding those in app/assets/stylesheets//bulma-overrides.scss to either match the gray when unstacked or use one of the reds we use elsewhere?  Something like `a { color: $primary }` maybe?

**Comment by phallstrom** [comment]
@Talha5 went ahead and merged this. I'll sneak in the color momentarily.  Thanks!

**Pull request merged**